.github/workflows/apisec-scan.yml

```
@@ -74,6 +74,8 @@ permissions:
 jobs:
 
   Trigger_APIsec_scan:
+    # Skip if APIsec credentials are not configured
+    if: ${{ secrets.apisec_username != '' && secrets.apisec_password != '' }}
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)